test/spark.js

```diff
@@ -69,6 +69,52 @@ test('getRetrieval', async () => {
   ])
 })
 
+test('testHeadRequest', async () => {
+  const requests = []
+  const spark = new Spark({
+    fetch: async (url, { method }) => {
+      requests.push({ url: url.toString(), method })
+      return {
+        status: 200
+      }
+    }
+  })
+  const stats = {}
+  await spark.testHeadRequest('/dns/frisbii.fly.dev/tcp/443/https', KNOWN_CID, stats)
+  assertEquals(stats.headStatusCode, 200)
+  assertEquals(requests, [{ url: `https://frisbii.fly.dev/ipfs/${KNOWN_CID}?dag-scope=block`, method: 'HEAD' }])
```

Comment:
Have you considered the feedback in https://github.com/CheckerNetwork/frisbii-on-fly/issues/3#issuecomment-2667253367?

> Once fixed, you may want to test with explicit content type: `HEAD /ipfs/cid?format=raw` + pass `Accept: application/vnd.ipld.raw` to ensure your requested [trustless block responses](https://specs.ipfs.tech/http-gateways/trustless-gateway/) and avoid issues if the other end supports deserialized responses as well.
> 
> There is a risk of false positive if deserialized response happens to fit in a single block, so choosing CID of a chunked data may be a good idea.

Should we implement this suggestion as part of the work adding HEAD request checking to Spark? If not, can you create a follow-up issue please?

Reply:
I was wondering about this. We're passing `?dag-scope=blog`, my understanding is this implies a crustless block response. Do you know more here? Just adding the Accept header doesn't hurt though, I'll do this

Reply:
328c30d